roles/ansible-keylime-tpm20/tasks/packages.yml

```diff
@@ -33,6 +33,16 @@
         - procps-ng
         - tpm2-tss
         - tpm2-tools
-        - tpm2-abrmd
```

Comment:
Just so I'm clear, swtpm replaces tpm2-abrmd? Does swtpm-tools replace tpm2-tools as well?

Reply:
the kernel tpm resource manager can do what tpm2_abrmd  and its less prone to breaking.

Reply:
So this means we still *do* needs tpm2-tools then, right?
(this would be consistent with the proposed state of `packages.yml`)